src/curl.c

```diff
@@ -165,19 +165,19 @@ int swupd_curl_init(void)
 	struct stat st;
 
 	if (curl) {
-		error("curl has already been initialized.\n");
+		error("curl has already been initialized\n");
 		return -1;
 	}
 
 	curl_ret = curl_global_init(CURL_GLOBAL_ALL);
 	if (curl_ret != CURLE_OK) {
-		fprintf(stderr, "Error: failed to initialize CURL environment\n");
+		error("failed to initialize CURL environment\n");
```

Comment:
to be consistent with all other curl messages you could change this to:
"curl - failed to initialize environment\n"